_partners/infino.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Infino
+name_long: 'Infino AI'
+logo: '/assets/media/partners/infino.png'
+link: 'https://infino.ai'
+logo_large: '/assets/media/partners/infino/infino_logo.png'
+description: 'Infino offers a schemaless, serverless managed OpenSearch service. Founded by former AWS OpenSearch engineers, we have replaced Lucene with a next-gen proprietary engine optimized for Observability use cases and internal developer platforms. 
+<br/><br/>
+We offer everything you have come to expect from OpenSearch except we are cheaper, faster, and with more observability features. Store logs, metrics, traces, code, tickets, etc. in one index. Query all your data at once via languages you know and love: Natural language, Query DSL, PromQL, SQL, PPL, etc. No snapshots, no mapping explosions, no astronomical bills, no slow aggregations, no cluster management, no minimum OCUs. 
+<br/><br/>
+Save money, save time, and keep your commitments to your customers.'
+business_type: ISV
+region: 'North America, South America, Europe, Middle East, Africa, Asia Pacific, Australia'
+contact: 'tech@infino.ai'
+opensearch_tech: 'Observability'
+industries: 'Software and Technology'
+main_office_location:  |
+   Infino AI Inc.
+   19266 Coastal Hwy
+   Unit 4-250
+   Rehoboth Beach, DE 19971
+products:
```

Comment:
I think this line might be repeated on accident.

Reply:
@vinaykakade : I don't think I have access to push a quick change to your `main` branch - feel free to revise your PR when you get a chance and we'll push you live!

Reply:
Hi @nateynateynate - thanks you for pointing this out. Fixed. Also, have given you push access to my fork.